src/server/lib/utils/content-utils.ts

```
@@ -0,0 +1,32 @@
+
+export const getLocalLessonUrl = (contentKey: string) => {
+	if (typeof process.env.STORAGE_ROOT_URL !== 'string') {
+		throw new Error('STORAGE_ROOT_URL is not defined');
+	}
+
+	return path.join(process.env.STORAGE_ROOT_URL, contentKey);
+}
+
+export const getLessonContent = async (contentKey: string) => {
+	// read from local file if we are in the dev environment
+	if (0 && process.env.NODE_ENV === 'development') {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)